packages/node-cli/src/utils/CommandUtils.js

```diff
@@ -38,6 +38,13 @@ class CommandUtils {
 	quoteString(string) {
 		return `"${string}"`;
 	}
+
+	unquoteString(stringValue) {
+		if (stringValue.length > 1 && stringValue.startsWith('"') && stringValue.endsWith('"')) {
```

Comment:
Why >1 ?

Reply:
I was thinking on `"`.

Reply:
I don't know if that can happen and if it does the result would be weird because the function will return the double quotes.... Anyway it would be a very rare case so I wouldn't mind too much :)